.storybook/reset-decorator.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+import { CssReset } from '../src'
+
+const CssResetWrapper = fn => (
+    <div style={{ margin: '10px' }}>
```

Comment:
It's completely irrelevant, but you could just write `margin: 10`, instead of `margin: '10px'`. For the lazy people ;) No need to change anything here!